server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -317,6 +329,39 @@ public String executor() {
         }
     }
 
+    TransportResponse handleExtensionRequest(ExtensionRequest extensionRequest) throws Exception {
+        // Read enum
+        switch (extensionRequest.getRequestType()) {
+            case REQUEST_EXTENSION_CLUSTER_STATE:
+                ClusterStateResponse clusterStateResponse = new ClusterStateResponse(
```

Comment:
I don't see the value of temporarily using another variable here vs. the original `return new ClusterStateResponse(...)`.   Similar comment for the other cases.  But the new code is inline.

Is there a reason this is preferred?  Or did this perhaps start from a branch before the other code got trimmed down?

Reply:
I didn't change anything about this, only added a switch statement instead of if/else, since that wasn't there when I first raised the PR

Reply:
You can periodically re-sync from main.  The shorter lines and javadoc are in the diff a little lower in this file... should be a copy-paste exercise to move them.